src/metabase/email/alert_you_were_added.mustache

```diff
@@ -0,0 +1,7 @@
+{{> metabase/email/_header }}
+  <div style="padding: 2em 4em; border: 1px solid #dddddd; border-radius: 2px; background-color: white; box-shadow: 0 1px 2px rgba(0, 0, 0, .08);">
```

Comment:
It seems silly we have to do the "card" styles for all of these templates. We should consider moving that to a shared partial.

Reply:
True!

Reply:
yeah 👍  there is a lot of repetition going on between the different templates, it would be nice to consolidate them into a partial if possible